pkg/mapper/conditions_changed.go

```diff
@@ -0,0 +1,12 @@
+package mapper
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+)
+
+// IgnoringTimeChangesInToolchainConditions wraps the provided event handler such that it only reacts
+// to updates discarding the changes in the condition timestamps. Only updates are affected, all other
+// event types are always forwarded.
+func IgnoringTimeChangesInToolchainConditions(eventHandler handler.EventHandler) handler.EventHandler {
```

Comment:
minor - should we keep the name more generic ? Just `IgnoringTimeChangesInConditions`, as i think those are not specific/custom toolchain conditions using these fields.

Reply:
Good suggestion, but I redid the implementation according to @MatousJobanek comments, so this actually no longer applies.